app/page.tsx

```diff
@@ -1,8 +1,17 @@
+'use client';
+
+import { useRouter } from "next/navigation";
+import { useEffect } from "react";
+
 const HomePage = () => {
+  const router = useRouter();
+
+  useEffect( () => {
+    router.push('/dashboard');
+  },[router]);
+
   return (
-    <div className=''>
-      This is <strong>Home</strong> page
-    </div>
+     <p>Redirecting...</p>
```

Comment:
We will not show any text while redirecting, so put here a fragment only `<></>`